src/cbc_sdk/rest_api.py

```diff
@@ -59,6 +59,16 @@ def _perform_query(self, cls, **kwargs):
         else:
             raise ApiError("All Carbon Black Cloud models must provide _query_implementation")
 
+    @property
+    def org_urn(self):
+        """
+        Returns the URN of the current organization.
```

Comment:
I would modify this docstring to be `Returns the URN based on the configured org_key.`

Reply:
So modified.